### PR TITLE
Unittest business logic of gRPC Client + test Services via grpc_cli

### PR DESCRIPTION
**Pull request by lukasmittag**
Unittests for the gRPC Services and testing the logic of the gRPC Client

### CODE REVIEW

File: test/unit-test/kuksa_grpc_client_mock.cpp [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (wenwenchenbosch) [inline test/unit-test/kuksa_grpc_client_mock.cpp:1]: empty file?

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -48,6 +48,8 @@ set(_GRPC_CPP_PLUGIN_EXECUTABLE $<TARGET_FILE:grpc_cpp_plugin>)
 set(_REFLECTION grpc++_reflection)
 set(_PROTOBUF_PROTOC $<TARGET_FILE:protoc>)
 
+include(GoogleTest)
```
Comment (wenwenchenbosch) [inline src/CMakeLists.txt:51]: googletest needed?

**Review by wenwenchenbosch — COMMENTED** [review]
I just see, the PR is a draft pr.

File: src/CMakeLists.txt (hunk 2 above)
Comment (lukasmittag) [reply src/CMakeLists.txt:51]: no, I will fix. I tried something.

**Review by wenwenchenbosch — APPROVED** [review]

**Pull request merged**